src/osd/osd_types.h

```diff
@@ -599,7 +599,8 @@ class coll_t {
     }
     return false;
   }
-
+  spg_t* get_pgid() { return &pgid; }
```

Comment:
You can remove this change because is_pg(spg_t *) validates that the collection is TYPE_PG and if so fills in the spg_t passed by pointer.